src/cli/cli.js

```diff
@@ -100,6 +100,8 @@ async function runTests (argParser) {
 
     try {
         failed = await runner.run(opts);
+
+        await marketing.showMessageWithLinkToTestCafeStudio();
```

Comment:
Let's not to write this message in stdout for parsable reporters like xunit, json etc.
So I suggest to output these messages only for 'spec', 'list', 'minimal' reporters.